modules/ROOT/examples/live-demos/uploadcare/example.js

```diff
@@ -0,0 +1,21 @@
+tinymce.init({
+    selector: "textarea",
+    plugins: [ "uploadcare", "code", "link", "preview", "lists" ],
+    uploadcare_public_key: '<your-public-key>',
+    uploadcare_store_type: 'temporary',
```

Comment:
It's not likely that customers will configure this so maybe we skip that from being in the demo if they want to change the default for some odd reason then they can look that up in the docs.

Reply:
Yeah agreed, I will remove from the example.js file.